denylist/src/denylist.rs

```diff
@@ -150,6 +161,13 @@ fn public_key_hash<R: AsRef<[u8]>>(public_key: R) -> u64 {
     hasher.finish()
 }
 
+fn edge_hash(edge_key: (&[u8], &[u8])) -> u64 {
+    let mut hasher = XxHash64::default();
+    hasher.write(edge_key.0);
```

Comment:
I assume the denylist filter is being updated or has already to include these edge keys ?